packages/e2e-tests/src/fixture/localStorageInitializer.ts

```diff
@@ -97,14 +56,33 @@ class LocalStorageInitializer {
     await localStorageManager.setItem('showPinExtension', 'false');
   };
 
-  initialiseBasicLocalStorageData = async (
-    walletName: string,
-    chainName: 'Preprod' | 'Preview' | 'Mainnet'
-  ): Promise<void> => {
+  async initializeAppSettings(): Promise<void> {
+    const network = extensionUtils.getNetwork().name;
+    const appSettings = `{"chainName":"${network}"}`;
+    await localStorageManager.setItem('appSettings', appSettings);
+  }
+
+  initialiseBasicLocalStorageData = async (walletName = 'TestAutomationWallet'): Promise<void> => {
+    // Pause fix for flaky tests where local storage keys are disappearing when executed right after opening the extension
+    await browser.pause(500);
```

Comment:
I know this wasn't a change in this PR so not a change request, but want to discuss the use of `browser.pause`: imho it should only be used as last resort, as it indicates a race condition, which can be flaky even with the pause. Also it can make tests slower than necessary. Would this be possible to implement here, for example?
```suggestion
    // Pause fix for flaky tests where local storage keys are disappearing when executed right after opening the extension
    await localStorageReady();
```

Reply:
I totally agree about browser.pause and we generally stick to this rule when writing tests. There are some rare exceptions and I believe this is one of them :) I think that this particular one could be removed. I will do that in a separate PR.